create_graphics.py

```diff
@@ -557,15 +558,14 @@ def graphics_driver(cla):
                             )
 
             # Zip png files and remove the originals in a subprocess
-            if cla.zip_dir:
-                for tile, zipf in zipfiles.items():
-                    png_files = glob.glob(os.path.join(workdir, f'*_{tile}_*{fhr:02d}.png'))
-                    zip_proc = Process(group=None,
-                                       target=create_zip,
-                                       args=(png_files, zipf),
-                                       )
-                    zip_proc.start()
-                    zip_proc.join()
+            for tile, zipf in zipfiles.items():
+                png_files = glob.glob(os.path.join(workdir, f'*_{tile}_*{fhr:02d}.png'))
+                zip_proc = Process(group=None,
+                                   target=create_zip,
+                                   args=(png_files, zipf),
+                                   )
+                zip_proc.start()
+                zip_proc.join()
```

Comment:
This section needs to be inside an if statement: `if cla.zip_dir` as it was before.

Reply:
Thanks, I must have missed that.  I added it back in.